new-lamassu-admin/src/components/editableTable/Row.js

```diff
@@ -192,7 +192,7 @@ const ERow = ({ editing, disabled, lastOfGroup }) => {
 
   const classes = useStyles()
 
-  const shouldStripe = stripeWhen && stripeWhen(values) && !editing
+  const shouldStripe = stripeWhen && stripeWhen(values)
```

Comment:
Have you make sure that other places that contain stripe aren't being broken by this change?